sample/test/service/event_test.dart

```diff
@@ -2,13 +2,27 @@ import 'dart:convert';
 
 import 'package:GTUBT/service/event.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:GTUBT/models/api_response.dart';
 import 'package:GTUBT/models/event.dart';
+import 'package:http/http.dart';
 
 void main() {
+  test('Update Event with an ID', () async {
```

Comment:
bu test gercekten istek mi atiyor? http call yapip yapamadigimizin testi aslinda bunlar. bunun yerine response'lar mocklanmali ve test oyle yapilmali. Bunu baska bir is icinde degerlendirelim

Reply:
Evet istek atıyor. DB mizi de değiştiriyor. O mock işi mantıklı olur sıkıntı yaşamadan ona geçelim

Reply:
bu pr'da bunu iceri alacagim ama bunun icin is acalim ve bunlari refactor yapalim